src/cloud-api-adaptor/docs/SecureComms.md

```diff
@@ -0,0 +1,125 @@
+- Worker Node establishes an "Attestation Phase" SSH channel with the Peer Pod VM
+- Peer Pod VM attests and obtain keys from Trustee. The Peer Pod then signal to the Worker Node that the "Attestation Phase" has ended and terminates the "Attestation Phase" SSH channel.
+- Worker Node establishes an "Kubernetes Phase" SSH channel with the Peer Pod VM  - Both sides verify the identity of the other side using the keys delivered via Trustee.
+
+Once the "Kubernetes Phase" SSH channel is established, Secure Comms connects the `Runtime shim` to the `Kata Agent` and may also connect other services required by the Peer Pod VM or by containers running inside the Peer Pod.
+
+See [Secure Comms Architecture Slides](./SecureComms.pdf) for more details.
+
+## Setup
+
+### Deploy CAA
+Use any of the option for installing CAA depending on the cloud driver used.
+
+### Deploy KBS-Operator
+Deploy KBS-Operator by following instructions at [KBS Operator Getting Started](https://github.com/confidential-containers/kbs-operator?tab=readme-ov-file#getting-started).
+
+Copy the kbs-client secret from the `kbs-operator-system` namespace to the `confidential-containers-system` ns. This can be done using:
+
+```sh
+kubectl get secret kbs-client -n kbs-operator-system -o json|jq --arg ns "confidential-containers-system" 'del(.metadata["creationTimestamp","resourceVersion","selfLink","uid","annotations"]) | .metadata.namespace |= $ns' |kubectl apply -f -
+```
+
+For a testing environment, you may need to change the policy of the KBS and AS using the KBS Client to allow all or fit your own policy. One way to do that is:
+
+```sh
+kubectl -n kbs-operator-system exec deployment/kbs-deployment --container as -it -- /bin/bash
+        apt update
+        apt install vim
+        vim /opt/confidential-containers/attestation-service/opa/default.rego // replace to `default allow = true`
+
+kubectl -n kbs-operator-system exec deployment/kbs-deployment --container kbs -it -- /bin/bash
+        apt update
+        apt install vim
+        vim /opa/confidential-containers/kbs/policy.rego // replace to `default allow = true`
+```
+
+### Build a podvm that enforces Secure-Comms
+
+Change the `src/cloud-api-adaptor/podvm/files/etc/systemd/system/agent-protocol-forwarder.service` to include:
+```sh
+ExecStart=/usr/local/bin/agent-protocol-forwarder -kata-agent-namespace /run/netns/podns -secure-comms -kata-agent-socket /run/kata-containers/agent.sock $TLS_OPTIONS $OPTIONS
+```
+
+You may also include additional Inbounds and Outbounds configurations to the Forwarder using the `-secure-comms-inbounds` and `-secure-comms-outbounds` flags. See more details regarding Inbounds and Outbounds below.
+
+Once you changed `podvm/files/etc/systemd/system/agent-protocol-forwarder.service`, you will need to [rebuild the podvm](./../podvm/README.md).
+
+
+### Activate CAA Secure-Comms feature
+
+Add to the `peer-pods-cm` config map at the `confidential-containers-system` namespace:
```

Comment:
Maybe it would be helpful to give the command: `kubectl edit cm peer-pods-cm -n confidential-containers-system` here?